data-raw/stress_test_arguments.R

```diff
@@ -2,8 +2,8 @@
 stress_test_arguments <- tibble::tribble(
                       ~name,       ~type, ~default,               ~allowed,       ~min,       ~max,
```

Comment:
please run devtools::test or check and update the snapshot test.

Reply:
updated the one for the arguments, some of the other fails may not be actual errors. Will confirm on Monday